dev-client/src/screens/HomeScreen/components/TemporarySiteCallout.tsx

```diff
@@ -68,10 +77,14 @@ export const TemporarySiteCallout = ({coords, closeCallout}: Props) => {
           value={TEMP_ECO_SITE_PREDICTION.toUpperCase()}
         />
         <Divider />
-        <CalloutDetail
-          label={t('site.elevation').toUpperCase()}
-          value={TEMP_ELEVATION.toUpperCase()}
-        />
+        {siteElevation ? (
```

Comment:
Wouldn't this leave the spinner in place for sites that happen to be at zero sea level? I recognize that this may be very unlikely in practice but it still seems like an incorrect implementation.

Reply:
Since this is a string and not a float, `"0"` an `"0.0"` are truthy, so the spinner would go away. I could use `siteElevation !== ''` if you'd prefer.

Reply:
That might be more clear - wasn't obvious to me that this was a string.